airflow/dag_processing/manager.py

```diff
@@ -599,7 +599,10 @@ def _run_parsing_loop(self):
 
             if self.standalone_dag_processor:
                 self._fetch_callbacks(max_callbacks_per_loop)
-            self._scan_stale_dags()
+
+            # this variable gives us flexibility to purge stale dags or not.
```

Comment:
This is obvious from actual logic; the comment is quite redundant.

Reply:
Indeed, I left this comment because it seems to me that the function name is not explicit enough in the sense that it does not only scan but also deletes the dags.

Reply:
But can we rename it like this? 
For exemple: _scan_stale_dags > _scan_and_deletion_stale_dags

Reply:
Replace deletion with _delete_ I think it makes sense. Or just *delete (or purge) stale DAGs* (since obviously you need to find them to delete them).